composer.json

```diff
@@ -5,45 +5,43 @@
     "homepage": "https://github.com/silverstripe/recipe-kitchen-sink",
     "license": "BSD-3-Clause",
     "require": {
-        "php": "^7.4 || ^8.0",
-        "silverstripe/recipe-plugin": "^1",
-        "silverstripe/installer": "4.x-dev",
-        "silverstripe/recipe-authoring-tools": "1.x-dev",
-        "silverstripe/recipe-blog": "1.x-dev",
-        "silverstripe/recipe-collaboration": "1.x-dev",
-        "silverstripe/recipe-form-building": "1.x-dev",
-        "silverstripe/recipe-reporting-tools": "1.x-dev",
-        "silverstripe/recipe-services": "1.x-dev",
-        "silverstripe/recipe-content-blocks": "2.x-dev",
-        "silverstripe/recipe-solr-search": "2.x-dev",
-        "silverstripe/recipe-ccl": "2.x-dev",
-        "silverstripe/developer-docs": "4.x-dev",
-        "silverstripe/registry": "2.x-dev",
-        "silverstripe/totp-authenticator": "4.x-dev",
-        "silverstripe/mfa": "4.x-dev",
-        "silverstripe/crontask": "2.x-dev",
-        "silverstripe/gridfieldqueuedexport": "2.x-dev",
-        "silverstripe/ldap": "1.x-dev",
-        "silverstripe/textextraction": "3.x-dev",
-        "silverstripe/realme": "4.x-dev",
-        "silverstripe/ckan-registry": "1.x-dev",
-        "silverstripe/webauthn-authenticator": "4.x-dev",
-        "silverstripe/security-extensions": "4.x-dev",
-        "silverstripe/subsites": "2.x-dev",
-        "silverstripe/html5": "2.x-dev",
-        "cwp/starter-theme": "3.x-dev",
-        "cwp/agency-extensions": "2.x-dev",
-        "cwp/watea-theme": "3.x-dev",
-        "cwp/cwp": "2.x-dev",
-        "cwp/cwp-pdfexport": "1.x-dev",
-        "dnadesign/silverstripe-elemental-userforms": "~3.2.1@stable",
-        "symbiote/silverstripe-multivaluefield": "5.x-dev",
-        "tractorcow/silverstripe-fluent": "4.x-dev"
+        "php": "^8.1",
+        "silverstripe/recipe-plugin": "^2",
```

Comment:
Won't this need to be the x-dev constraint as per https://github.com/silverstripe/silverstripe-installer/pull/340? Or will the fact that it's x-dev in installer take precedence already?

Reply:
Ah yup, have changed to 2.x-dev and put a note on https://github.com/silverstripe/silverstripe-installer/issues/341 to also handle recipe-kitchen-sink